packages/fxa-auth-server/lib/senders/emails/templates/passwordChanged/index.mjml

```diff
@@ -16,5 +16,5 @@
   </mj-column>
 </mj-section>
 
-<%- include('/partials/location/index.mjml') %>
+<%- include('/partials/location/index.mjml', { cssClass: 'text-body-grey-no-margin' }) %>
```

Comment:
Hmm, so this is a new pattern, and I don't mind it, but I kinda wonder if instead of modifying the margin here perhaps we could modify the margins around it? I think we already have classes for that?

Reply:
You're supposed to say "gosh this is cool!" 😉

For this particular case, I didn't see a way around it. We either need to remove the margin-bottom of the location partial (which is what I did), or reduce the margin-top of the automated email footer text.

With change:
<img width="554" alt="image" src="https://user-images.githubusercontent.com/13018240/132871150-dae763f4-8a67-489b-af0f-1cc3aa3328e9.png">

Without change:
<img width="536" alt="image" src="https://user-images.githubusercontent.com/13018240/132871428-c9101295-9cbc-4663-a77e-abea16a67376.png">

I could see us using this pattern in other templates too, though I suppose I can't say for sure.

Reply:
Sounds good! And is _IS_ cool!